code/style.css

```diff
@@ -1,3 +1,107 @@
+:root {
+  --gblue: rgb(87, 166, 255);
+}
 body {
-  background: #FFECE9;
```

Comment:
In the css I think it would be nice with an empty line between. Easier to read. It gets a bit hard to seperate it now.